### DISCUSSION TITLE
Using a redis connection (not ioredis)

### DISCUSSION THREAD

**Discussion by avasilic** [taskforcesh/bullmq#2273] [Q&A]
Hi there, how do you use a connection from the [redis](https://www.npmjs.com/package/redis) package, not `ioredis`?

**Comment by manast** [comment]
Only ioredis is currently supported. Previously we supported the redis package (before redis.com took over it), and it had a number of limitations. We are probably not going to support both as it is quite a lot of work to do and ioredis works just fine.
